clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```
@@ -663,6 +676,8 @@ public void close(Duration timeout) {
         long now = time.milliseconds();
         long newHardShutdownTimeMs = now + waitTimeMs;
         long prev = INVALID_SHUTDOWN_TIME;
+        final long reporterTimeoutMs = waitTimeMs;
+        clientTelemetryReporter.ifPresent(reporter -> reporter.initiateClose(reporterTimeoutMs));
```

If this operation can fail, would we want just let it bubble up from `close()` as we do right now, of would it make sense to catch-log-and-swallow to move forward this `close()`?

---

The `ClientTelemetryReporter.initiateClose` does the catch-log-and-swallow if there's exception in this method